packages/monorepo-scripts/src/contract_artifacts_copy.ts

```diff
@@ -0,0 +1,54 @@
+import * as artifacts from '@0x/contract-artifacts';
+const artifactsToPublish = Object.keys(artifacts);
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { utils } from './utils/utils';
+
+const pkgNames = process.argv.slice(2);
+const contractsDirs = [];
+for (const pkgName of pkgNames) {
+    if (pkgName.indexOf('/contracts-') === -1) {
+        throw new Error(`Invalid package name: [${pkgName}]. Contracts packages must be prefixed with 'contracts-'`);
+    }
+    contractsDirs.push(pkgName.split('/contracts-')[1]);
+}
+
+const contractsPath = path.join(__dirname, '../../../contracts');
+const allArtifactPaths = [];
+for (const dir of contractsDirs) {
+    const artifactsDir = path.join(contractsPath, dir, 'generated-artifacts');
+    if (!fs.existsSync(artifactsDir)) {
+        continue;
+    }
+    const artifactPaths: string[] = fs
+        .readdirSync(artifactsDir)
+        .filter(artifact => {
+            const artifactWithoutExt = artifact.split('.')[0];
+            return artifactsToPublish.includes(artifactWithoutExt) && !shouldSkip(dir, artifactWithoutExt);
+        })
+        .map(artifact => path.join(artifactsDir, artifact));
+    allArtifactPaths.push(...artifactPaths);
+}
+
+for (const _path of allArtifactPaths) {
+    const fileName = _path.split('/').slice(-1)[0];
+    const targetPath = path.join(__dirname, '../../contract-artifacts/artifacts', fileName);
+    const targetPathPython = path.join(__dirname, '../../../python-packages/contract_artifacts/src/zero_ex/contract_artifacts/artifacts', fileName);
+    fs.copyFileSync(_path, targetPath);
+    fs.copyFileSync(_path, targetPathPython);
+    utils.log(`Copied from ${_path} to ${targetPath}`);
+}
+utils.log(`Finished copying contract-artifacts. Remember to lint artifacts before using abi-gen.`);
```

Comment:
```suggestion
utils.log(`Finished copying contract-artifacts.`);
```

The transform happens automatically now right?

Reply:
Not necessarily. The yarn shortcut runs `copy && transform`, but this script only does `copy`, so I think we should keep this log line in case the script is used alone.

The resulting output looks like this:
```
[17:24:58]contract-artifacts (feature/prepublish-contract-artifacts)$ yarn artifacts_update
yarn run v1.9.4
$ yarn artifacts_copy && yarn artifacts_transform
$ node lib/src/copy.js ${npm_package_config_contractsPackages}
Finished copying contract-artifacts. Remember to transform artifacts before publishing or using abi-gen
$ node lib/src/transform.js ./artifacts && prettier --write ./artifacts/*.json &&  cp -r ./artifacts/ ../../python-packages/contract_artifacts/src/zero_ex/contract_artifacts/artifacts/
Deleting forbidden properties from artifacts in ./artifacts. Output to ./artifacts
artifacts/AssetProxyOwner.json 117ms
artifacts/Coordinator.json 37ms
artifacts/CoordinatorRegistry.json 22ms
artifacts/DummyERC20Token.json 27ms
artifacts/DummyERC721Token.json 37ms
artifacts/DutchAuction.json 22ms
artifacts/ERC20Proxy.json 27ms
artifacts/ERC20Token.json 25ms
artifacts/ERC721Proxy.json 23ms
artifacts/ERC721Token.json 19ms
artifacts/Exchange.json 74ms
artifacts/Forwarder.json 20ms
artifacts/IAssetProxy.json 9ms
artifacts/IValidator.json 8ms
artifacts/IWallet.json 7ms
artifacts/MultiAssetProxy.json 12ms
artifacts/OrderValidator.json 21ms
artifacts/WETH9.json 14ms
artifacts/ZRXToken.json 13ms
✨  Done in 2.54s.
[17:25:12]contract-artifacts (feature/prepublish-contract-artifacts)$ 

```

Reply:
Ok